### ISSUE TITLE
Big Memory-leak issues on Desktop

### ISSUE THREAD

**Issue by SuggestionGuy** [brave/brave-browser#9896]
On both my Desktop and my Laptop I have a problem with Brave completely sucking up all the RAM on my computer until they hit 95-98% usage. My first thought is always that it's browser extensions, but when I use other devices which do not have any browser extensions (especially ad-blocking ones), I find that my RAM is still being gobbled up. 

I do not have malware or crypto-miners on my computers. Even FUD. 
I'm very meticulous with what I download.

Task manager specifically shows that it's Brave causing issues. I have 16GB of RAM in my computer and I'll see my Memory at 96% and I'll close brave and it instantly goes down to 40-65%. 
 
It seems I'm not the only one having this issue either. A quick search for; "Brave memory issues" pulls up a Reddit post that's 5-months old of someone having the same issue as me. 
It's gotten so bad that I sometimes can't even use Brave. It uses far more resources than it's competitors. 

I may be an idiot here. 
I keep a lot of tabs open (like 30-40) and I have Brave set to remember what I last had opened if I close the browser. I know this is a huge part of the strain here. But when I do this with ANY other browser. I do not get results like this. It works just fine; no memory issues. 

I hope someone can provide insight or help fix the issue if the issue doesn't lay between the headphones.

**Comment by bsclifton** [comment]
@SuggestionGuy do you have more information about your setup? What OS are you running? What version of Brave are you running?

Something that would be helpful too when you experience the problem: go up to the hamburger menu (top right, three lines) and click, from menu pick `More tools` > `Task Manager`. This should give a breakdown of what's running in Brave. It would be helpful to have a screenshot of this or just look for what is taking the most memory and share that

**Comment by SuggestionGuy** [comment]
- Windows 7
- Always running the newest version of brave: 
![image](https://user-images.githubusercontent.com/62922600/83085290-23b3c300-a051-11ea-8da4-d6ffca90cba8.png)  
Also: 
![image](https://user-images.githubusercontent.com/62922600/83085264-11d22000-a051-11ea-909f-c7ce0da80569.png)
![image](https://user-images.githubusercontent.com/62922600/83085409-77bea780-a051-11ea-9750-63d3f17df514.png)

(Although right now it's not throttling my memory). I'll do it again when it is. 

I added a plugin called "OneTab" which puts all my tabs into just one tab hosted on the apps server. And it's been a great help. Same with moving some of the tabs onto Chrome.

**Comment by SuggestionGuy** [comment]
Okay here was a big spike that slowed everything down. 
![image](https://user-images.githubusercontent.com/62922600/83207707-7a84cf80-a119-11ea-8f3b-4aa01c13464c.png)

**Comment by rebron** [comment]
@SuggestionGuy Are you still running into this issue with the latest 1.10.x release? We'll also be releasing 1.11.x next week, this may be fixed with the latest chromium updates.

**Comment by SuggestionGuy** [comment]
> @SuggestionGuy Are you still running into this issue with the latest 1.10.x release? We'll also be releasing 1.11.x next week, this may be fixed with the latest chromium updates.

Yes, I'm still having this issue in "Version 1.11.97 Chromium: 84.0.4147.89 (Official Build) (64-bit)" 
I apologize for my late reply. I had it on a sticky note. I just never got around to it. I intend to be quicker in future responses.

**Comment by LefterisJP** [comment]
I am experiencing very serious memory leak issues in Desktop in Archlinux very recently. Before it was fne.

Since the following version: `Version 1.14.84 Chromium: 85.0.4183.121 (Official Build) unknown (64-bit)`

After about 2 hours of brave running my system is out of memory and starts swapping.

**Comment by cyagon** [comment]
I have these problems too. Brave 1.17.73 on Windows 10.
One question: is it normal that Brave Rewards eats up 1,6GB of RAM
![brave_dAaKB2dEod](https://user-images.githubusercontent.com/32183089/100524536-4918b980-31b9-11eb-88a9-29a2518922ba.png)
?

**Comment by ahadda5** [comment]
![5 tabs and 26 Brave processes](https://user-images.githubusercontent.com/21275079/107267209-00ef0080-6a60-11eb-9263-ee69ab584acf.JPG)
It is not crashing however i believe it is using more memory than Chrome. 
those are 5 tabs and 26 brave processes! for 1.56GB of memory.  

I know this does not help much , but why do we have so many processes? 
1. tab is Discord which i know is memory heavy
2. one is github
3.  two tabs are tradingview , another heavy charting portal

What more info can i provide?
